scripts/linters/general_purpose_linter.py

```diff
@@ -241,7 +242,7 @@
             'Please do not define "any" types. You can refer '
             'https://github.com/oppia/oppia/wiki/Guide-on-defining-types '
             'if you\'re having trouble declaring types.'),
-        'excluded_files': (),
+        'excluded_files': ('core/templates/tests/unit-test-utils.ts'),
```

Comment:
Because of this line: https://github.com/oppia/oppia/pull/10427/files#diff-f0a8ab9c6df6553ef2f1e370755fccd7R82. The fail module is used in line 94 but for some reason, typescript is expecting it to be a string. But it shouldn't be a string otherwise we will run into errors during run time. The alternative is to use @ts-ignore. I don't which one is better.
What should I use @nishantwrp?

Reply:
I think @ts-ignore would be a better option.